src/aap_eda/settings/default.py

```diff
@@ -742,13 +742,13 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
     "PG_NOTIFY_DSN_SERVER", _DEFAULT_PG_NOTIFY_DSN_SERVER
 )
 
-WEBHOOK_BASE_URL = settings.get("WEBHOOK_BASE_URL", None)
-if WEBHOOK_BASE_URL:
-    WEBHOOK_BASE_URL = WEBHOOK_BASE_URL.strip("/") + "/"
+EVENT_STREAM_BASE_URL = settings.get("EVENT_STREAM_BASE_URL", None)
```

Comment:
Since webhooks are a subset of event streams and this variable would affect only to this subset, I think this variable should keep the existing name, or at least be renamed as "EVENT_STREAM_WEBHOOK_BASE_URL"

Just call it EVENT_STREAM_BASE_URL will introduce confusion in the future when we have more types of event streams.

Reply:
this is also relevant for the future changes required in the operator/install where a well self descriptive name is even more relevant.

Reply:
nevermind